concepts/05 Widgets/DropDownButton/00 Getting Started with DropDownButton/30 Populate the Drop-Down Menu.md

```diff
@@ -1,24 +1,24 @@
-To populate the drop-down menu, assign an array to the [items](/api-reference/10%20UI%20Widgets/dxDropDownButton/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/items/') option. Objects in the array should provide text, icons, and other menu item data. The data fields that match those listed in the [items](/api-reference/10%20UI%20Widgets/dxList/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/') section are automatically recognized ([`text`](/api-reference/_hidden/CollectionWidgetItem/text.md '/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/#text') and [`icon`](/api-reference/_hidden/dxListItem/icon.md '/Documentation/ApiReference/UI_Widgets/dxList/Configuration/items/#icon') in the code below). If you store item text in a field with a different name, use the [displayExpr](/api-reference/10%20UI%20Widgets/dxDropDownButton/1%20Configuration/displayExpr.md '/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/#displayExpr') option to map the field.
+To populate the drop-down menu, assign an array to the [items](/api-reference/10%20UI%20Widgets/dxDropDownButton/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/items/') option. Objects in the array should provide text, icons, and other menu item data. The data fields that match those listed in the [items](/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/items/) section are automatically recognized ([`icon`](/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/items/#icon) in the code below). To specify the field that stores item text, use the [displayExpr](/api-reference/10%20UI%20Widgets/dxDropDownButton/1%20Configuration/displayExpr.md '/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/#displayExpr') option.
```

Comment:
1) `To populate the drop-down menu, assign an array to the [items](/api-reference/10%20UI%20Widgets/dxDropDownButton/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/items/') option.`
Since this is the first sentence in the topic, we should reverse the order of the sentence. Can we change it to the following?

`Assign an array to the [items](/api-reference/10%20UI%20Widgets/dxDropDownButton/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxDropDownButton/Configuration/items/') option to populate the drop-down menu.`

2) `Objects in the array should provide text, icons, and other menu item data.`
Please rewrite without using "provide".